pkgs/top-level/all-packages.nix

```diff
@@ -1456,6 +1456,10 @@ with pkgs;
 
   parallel-rust = callPackage ../tools/misc/parallel-rust { };
 
+  scour = callPackage ../tools/graphics/scour {
+    inherit (python.pkgs) buildPythonApplication fetchPypi six;
```

Comment:
I would move this inside the package expression, it will make it easier to override python.

Reply:
Why use `python` instead of `python3`?